pages/Livesite/judging.js

```diff
@@ -55,6 +62,11 @@ export default ({ hackathons }) => {
   };
 
   const handleSave = async () => {
+    if (data.id) {
+      updateProject(activeHackathon, data);
+      setActiveModal('');
+      return;
+    }
     if (!Object.values(data).every((field) => !!field)) {
       // eslint-disable-next-line no-alert
       alert('All fields required');
```

Comment:
it won't let me comment on that line for some reason, but on lines 79-81 might be good to guard on
1. undefined
2. make sure to stripe whitespace after splitting (in case we end up with extra spaces)

Reply:
`if (!Object.values(data).every((field) => !!field))` should guard against undefined but i will strip whitespaces